executor/stale_txn_test.go

```diff
@@ -318,3 +248,21 @@ func (s *testStaleTxnSerialSuite) TestTimeBoundedStalenessTxn(c *C) {
 		failpoint.Disable("github.com/pingcap/tidb/store/tikv/injectSafeTS")
 	}
 }
+
+func (s *testStaleTxnSerialSuite) TestStalenessTransactionSchemaVer(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("create table t (id int primary key);")
+
+	schemaVer1 := tk.Se.GetSessionVars().GetInfoSchema().SchemaMetaVersion()
+	tk.MustExec("drop table if exists t")
+	schemaVer2 := tk.Se.GetSessionVars().GetInfoSchema().SchemaMetaVersion()
+	// confirm schema changed
+	c.Assert(schemaVer1, Less, schemaVer2)
+
+	tk.MustExec(`START TRANSACTION READ ONLY WITH TIMESTAMP BOUND EXACT STALENESS '00:00:01'`)
+	schemaVer3 := tk.Se.GetSessionVars().GetInfoSchema().SchemaMetaVersion()
+	// got an old infoSchema
+	c.Assert(schemaVer3, Less, schemaVer2)
+}
```

Comment:
I think we should sleep(1s) after we create job, and assert `schemaVer3` equals `schemaVer1` for this case.

Reply:
Done.